cypress/integration/full_run_simple.spec.js

```diff
@@ -151,7 +151,7 @@ describe('Full run through saying "no" to everything', function() {
     cy.continue()
   })
 
-  // NOTICE
+  // NOTICE OF ASSESSMENT
```

Comment:
lol sure

Reply:
yeah I think that made it in cuz I noticed it got overwritten on a rebase, so I just named it off the top of my head lol